ykman/cli/otp.py

```diff
@@ -89,6 +89,25 @@ def _confirm_slot_overwrite(controller, slot):
             abort=True)
 
 
+def explain_fips_status(is_fips_mode, slot1_programmed, slot2_programmed):
```

Comment:
Looks good, but should maybe be a private method?

Reply:
:+1: